packages/terra-folder-tree/src/FolderTree.jsx

```diff
@@ -1,26 +1,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { injectIntl } from 'react-intl';
-// import classNames from 'classnames/bind';
-// import styles from './FolderTree.module.scss';
+import classNames from 'classnames/bind';
 
-// const cx = classNames.bind(styles);
+import ActionHeader from 'terra-action-header';
+
+import FolderTreeItem from './subcomponents/FolderTreeItem';
+import styles from './FolderTree.module.scss';
+
+const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
-   * @private
-   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   * List of FolderTree.Items to be displayed as content within the FolderTree.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The title of the folder tree.
    */
-  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+  title: PropTypes.string.isRequired,
+  /**
+   * The heading level for the title of the folder tree.
+   */
+  headerLevel: PropTypes.number,
```

Comment:
Is this a value that we want to be configurable or should it be a constant value for all FolderTree components.

Reply:
Talked with @eawww, we want to keep the header level configurable since we don't know where on a page FolderTree will be used so consumers will need the option to set custom heading levels.